src/NServiceBus.Testing/Saga.cs

```diff
@@ -124,11 +126,11 @@ public Saga<T> ExpectReply<TMessage>(Func<TMessage, ReplyOptions, bool> check =
         /// <summary>
         /// Check that the saga replies with the given message type complying with the given predicate.
         /// </summary>
-        public Saga<T> ExpectReply<TMessage>(Func<TMessage, bool> check)
+        public Saga<T> ExpectReply<TMessage>(Func<TMessage, bool> check = null)
```

Comment:
I don't think we need to make that parameter optional since we already have `ExpectReply<TMessage>(Func<TMessage, ReplyOptions, bool> check = null)`?